modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -130,11 +130,10 @@ inline IE::TensorDesc toIE(const cv::Mat &mat, cv::gapi::ie::TraitAs hint) {
         const size_t height   = mat.size().height;
         const size_t width    = mat.size().width;
 
-        const size_t strideH  = mat.step[0];
-
-        IE::BlockingDesc bdesc({1, height, width, channels} /* dims */,
-                               {0, 2, 3, 1} /* order for NHWC */,
-                               0 /* offset */,
+        const size_t strideH  = mat.step1();
```

Comment:
@AsyaPronina we shouldn't consider the channel size in bytes, it's reason to use `step1()`

From OpenCV documentation:
```
Returns a normalized step.

The method returns a matrix step divided by Mat::elemSize1() . It can be useful to quickly access an arbitrary matrix element.
```